### PR TITLE
feat: add builder

### PR DESCRIPTION
**Pull request by cnt1992**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

File: packages/builder/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "engines": {
+    "npm": ">=3.0.0"
+  },
+  "dependencies": {
+    "react": "^16.8.2",
+    "react-dom": "^16.7.0",
+    "moment": "^2.24.0",
+    "@uform/react": "^0.0.9",
+    "@uform/utils": "^0.0.9",
+    "@uform/validator": "^0.0.9",
```
Comment (janryWang) [inline packages/builder/package.json]: 都改为0.1.0-alpha.3

**Comment by cnt1992** [comment]
> 超过两百行的代码都考虑一下拆分吧，不然可读性比较差

有一些代码是`style-component`样式比较多的原因，纯粹JS的我拆分一下

**Comment by janryWang** [comment]
@all-contributors please add @cnt1992 for developer

**Comment by janryWang** [comment]
@all-contributors please add @cnt1992 for design

**Pull request merged**